script/cmd/PluginManager/0x3169ad878021b87c9caa9b5cda740ff3ca270ce9/PluginManager

```diff
@@ -1 +1 @@
612a44575b8115612a33575b8115612a22575b8115612a11575b8115612a03575090565b63118a538960e01b14919050565b630c77631360e41b811491506129f9565b631128186d60e01b811491506129f2565b63236b075960e11b811491506129eb565b63bfd151c160e01b811491506129e4565b63031fb36160e21b811491506129dd565b63af87348360e01b811491506129d6565b632238633960e21b811491506129cf565b638a91b0e360e01b811491506129c8565b9190612aa481612b1f565b6001600160581b031991818316612ad5575050612ac082612b1f565b16156121c4576006612ad29201612b46565b50565b612ae29085939295612b46565b50612aec83612b1f565b16612af657505050565b612ad2926005612b1a9201906001600160581b031916600052602052604060002090565b612b46565b6bffffffffffffffffffffffff19815160601b1690602060ff60581b91015160581b161790565b906001600160581b0319612b5982612b1f565b16156121c45761219591612b76612b6f83612b1f565b9282612e55565b15612bd3575b60046002820191612ba18484906001600160581b031916600052602052604060002090565b612bab8154612597565b905501612bb88154612597565b9055906001600160581b031916600052602052604060002090565b612c506001820161207c84612bf18360008052602052604060002090565b5460581b92612c188261207c8689906001600160581b031916600052602052604060002090565b612c396121168388906001600160581b031916600052602052604060002090565b6121268261207c8360008052602052604060002090565b60038101612c5e8154612597565b9055612b7c565b9190612c7990612c73611201565b50612b1f565b60405192612c868461076d565b6064926064855260005b610c808110612d4657506001600160581b031980841615612d28575b6000935b85851080612d1d575b15612d0d57612d0161202f82612cd1612d079461223f565b612cdb898c61119f565b52612ce6888b61119f565b5086906001600160581b031916600052602052604060002090565b94612597565b93612cb0565b925050925061119c91845261223f565b508181161515612cb9565b9250612d4061202f8360008052602052604060002090565b92612cac565b602090612d51611201565b82828901015201612c90565b906001600160581b0319612d7082612b1f565b16156121c457612d7f90612b1f565b90612d8a8282612eef565b15611c0957612e4a61211683612db660029585906001600160581b031916600052602052604060002090565b54612e01600186019160581b61207c612de38585906001600160581b031916600052602052604060002090565b5460581b88906001600160581b031916600052602052604060002090565b6120f5612e228383906001600160581b031916600052602052604060002090565b5460581b61207c6120db61202f868a906001600160581b031916600052602052604060002090565b016127268154611c2d565b90612e5f90612b1f565b6001600160581b0319811615612e8f5760026121959201906001600160581b031916600052602052604060002090565b5050600190565b6000602091604051838101906301ffc9a760e01b8083526024820152602481526123a081610752565b6000602091604051838101906301ffc9a760e01b825263ffffffff60e01b6024820152602481526123a081610752565b906000805281602052604060002054906001600160581b031991828083169160581b1614928315612f58575b8315612f275750505090565b612f49929350600101906001600160581b031916600052602052604060002090565b5460581b161515388080612612565b925081612f798285906001600160581b031916600052602052604060002090565b5460581b16151592612f1b56fea26469706673582212203dc115b457d105434dd6f2fbae10e66d1ef974f77ef160b2312252e5b6f586e264736f6c63430008180033
```

Comment:
Why this update here? Is it just adding new line?

Reply:
Yes moving sub-directory under address now that we have multiple of it. Previously there wasn't the address subdirectory. 

Please let me know if you see the same in diff